Modules/_cryptmodule.c

```diff
@@ -4,7 +4,9 @@
 #include "Python.h"
 
 #include <sys/types.h>
-
+#ifdef __VXWORKS__
+#include <openssl/des.h>
+#endif
```

Comment:
Maybe just add
```c
#define crypt DES_crypt
```
if both functions have compatible interfaces?

Reply:
Yes. They have same prototypes. I've modified as you suggested. Thanks for your review.